R/surv_return.R

```diff
@@ -5,18 +5,19 @@
 #' @param surv (`matrix()`)\cr Matrix of predicted survival probabilities, rows are observations,
 #' columns are times. Number of columns should be equal to length of `times`.
 #' @param crank (`numeric()`)\cr Relative risk/continuous ranking. Higher value is associated
-#' with higher risk. If `NULL` then either set as `lp` if available or as the estimated
-#' survival expectation, computed by `colSums(surv)`.
+#' with higher risk. If `NULL` then either set as as `-response` if available or
+#' `lp` if available or as the sum of the cumulative hazard function (expected
```

Comment:
Really appreciate the PR. However this line isn't strictly speaking true. We only return `lp` for PH type models where higher lp means higher risk, whereas for AFT type models then we'd return `-lp`. Unsure how best to represent this here, I guess we could be explicit and just say that. So "`lp` if available for PH type models or `-lp` if available for AFT type models".

 What do you think?

When amended please re-run `devtools:: document` and commit the rd

Reply:
I see, an example of this is the case with `objective:aft` in the [xgboost survival learner](https://github.com/mlr-org/mlr3extralearners/blob/main/R/learner_xgboost_surv_xgboost.R#L206), where the conversion from `lp` to `-lp` would happen before calling this function (if it was needed to be called).

So in case of an AFT model, the user of this function has to make sure he/she passes `-lp` (the function doesn't know to set it to `-lp`). I will add a new commit to reflect that, would that be better?